modules/passkey/contracts/SafeWebAuthnSignerFactory.sol

```diff
@@ -43,8 +55,24 @@ contract SafeWebAuthnSignerFactory is ISafeSignerFactory {
         uint256 y,
         P256.Verifiers verifiers
     ) external view override returns (bytes4 magicValue) {
-        if (WebAuthn.verifySignature(message, signature, WebAuthn.USER_VERIFICATION, x, y, verifiers)) {
-            magicValue = ERC1271.MAGIC_VALUE;
+        address _singleton = SINGLETON;
+        bytes memory data = abi.encodePacked(
+            abi.encodeWithSignature("isValidSignature(bytes32,bytes)", message, signature),
+            x,
+            y,
+            verifiers
+        );
+
+        // solhint-disable-next-line no-inline-assembly
+        assembly {
+            let dataSize := mload(data)
+            let dataLocation := add(data, 0x20)
+
+            let success := staticcall(gas(), _singleton, dataLocation, dataSize, 0, 0)
+            if eq(success, 1) {
+                returndatacopy(magicValue, 0, returndatasize())
```

Comment:
I think this is incorrect. It will copy to offset `0`, the default value for a `bytes4` variable.

Reply:
good catch!